crates/holochain/src/sweettest/sweet_conductor.rs

```diff
@@ -457,6 +458,9 @@ impl SweetConductor {
             )
             .await?;
 
+        self.add_dna_files_to_sweet_conductor_cache(agent.clone(), dnas.clone())
```

Comment:
We need to know the agent for this to be able to construct the cell id to which a dna file is associated. In a follow-up PR I think I would like to push this adding of dna files to the cache into the `install_and_enable_app()` or more precisely into the `install_app()` method, to make sure they're also added to the cache if these latter methods get invoked directly.